src/Picqer/Financials/Exact/Connection.php

```diff
@@ -152,23 +193,118 @@ public function connect()
 
         return $client;
     }
-
+    
+    /**
+     * @return bool
+     */
+    public function pauseForMinuteLimitEnabled()
+    {
+        return $this->pauseForMinuteLimit;
+    }
+    
+    /**
+     * @return self
+     */
+    public function enablePauseForMinuteLimit()
+    {
+        $this->pauseForMinuteLimit = true;
+        return $this;
+    }
+    
+    /**
+     * @return self
+     */
+    public function disablePauseForMinuteLimit()
+    {
+        $this->pauseForMinuteLimit = true;
+        return $this;
+    }
+    
+    /**
+     * @return bool
+     */
+    public function pauseForMaintenanceEnabled()
+    {
+        return $this->pauseForMaintenance;
+    }
+    
+    /**
+     * @return self
+     */
+    public function enablePauseForMaintenance()
+    {
+        $this->pauseForMaintenance = true;
+        return $this;
+    }
+    
+    /**
+     * @return self
+     */
+    public function disablePauseForMaintenance()
+    {
+        $this->pauseForMaintenance = false;
+        return $this;
+    }
+    
+    /**
+     * Pauses the process until time limit is over
+     */
+    protected function pause()
+    {
+        if ($this->pauseForMinuteLimitEnabled()) {
+            if($this->minuteLimitExceeded()) {
+                sleep(60);
```

Comment:
These conditions about coding style are very different.

I think this should be fixed and please check other codes in this PR.

I think other codes should have the same problem.